test/test.ts

```diff
@@ -14,6 +14,6 @@ const restoreSandbox: () => void = sandbox.restore.bind(sandbox) as () => void;
 
 tabris._init(new ClientMock());
 
-require('../src').injector.jsxProcessor.unsafeBindings = 'error';
+(require('../src') as any).injector.jsxProcessor.unsafeBindings = 'error';
```

Comment:
why is it necessary to cast to `any` now?

Reply:
It always was, but compiler warnings in tests are currently not causing the build or tests to fail for some reason. I can look in to that afte the release.